.travis.yml

```diff
@@ -2,6 +2,7 @@ language: go
 go:
         - 1.7
         - 1.8
+        - 1.9
```

Comment:
1.10 was released few days ago, we should have that too

Reply:
Agree 👍